python/cuml/test/test_linear_model.py

```diff
@@ -119,6 +119,44 @@ def test_linear_models(datatype, X_type, y_type,
                            1e-1, with_sign=True)
 
 
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('X_type', ['ndarray'])
+def test_linear_models_default(datatype, X_type):
+    train_rows = np.int32(20*0.8)
+    X, y = make_regression(n_samples=(20), n_features=3,
+                           n_informative=2, random_state=0)
+    X_test = np.asarray(X[train_rows:, 0:]).astype(datatype)
+    X_train = np.asarray(X[0:train_rows, :]).astype(datatype)
+    y_train = np.asarray(y[0:train_rows, ]).astype(datatype)
+
+    # Initialization of cuML's linear and ridge regression models
+    cuols = cuLinearRegression()
+
+    curidge = cuRidge()
```

Comment:
Seems awkward that ridge and ols are mixed into one test. I'd separate so each is isolated.